source/routing/raptor.cpp

```diff
@@ -794,5 +951,108 @@ int RAPTOR::best_round(SpIdx sp_idx) {
     return -1;
 }
 
+std::string RAPTOR::print_jpps_from_sp() {
```

Comment:
Are those the following `print_*` functions only for debugging purposes ? 
They don't seem to be used anywhere.